b3sum: add page (#9838)
tldr-pages/tldr: pages/common/b3sum.md

# b3sum

> > A command line utility for calculating BLAKE3 hashes, similar to Coreutils tools like b2sum or md5sum.

Reviewer: ```suggestion
> Command line utility for calculating BLAKE3 hashes.
> It is similar to Coreutils tools like b2sum or md5sum.
```
Would this be better?

> More information: <https://github.com/BLAKE3-team/BLAKE3/tree/master/b3sum>.

- Calculate the BLAKE3 checksum for a file:

`b3sum {{path/to/file}}`

- Calculate BLAKE3 checksums for multiple files:

`b3sum {{path/to/file1}} {{path/to/file2}}`

- Calculate the BLAKE3 checksum from `stdin`:

`{{some_command}} | b3sum`

> - Create a file of BLAKE3 sums:

Reviewer: ```suggestion
- Create a file of BLAKE3 checksums:
```

`b3sum {{path/to/file}} > {{path/to/file.b3}}`

> - Read and verify a file of BLAKE3 sums:

Reviewer: ```suggestion
- Read and verify a file of BLAKE3 checksums:
```
